app/queries/subscription_content_change_query.rb

```diff
@@ -0,0 +1,53 @@
+class SubscriptionContentChangeQuery
+  def initialize(subscriber:, digest_run:)
+    @subscriber = subscriber
+    @digest_run = digest_run
+  end
+
+  def self.call(*args)
+    new(*args).call
+  end
+
+  def call
+    present
+  end
```

Comment:
might want to also have a `private_class_method :new` if self.call is the only route into this class but I know it's something we don't do a lot (or at all) at gov.uk